### DISCUSSION TITLE
WonderSwan and WonderSwan Color to be ther own folder/system

### DISCUSSION THREAD

**Discussion by Shadow322** [OnionUI/Onion#626] [Ideas]
Can we get them as ther own folder/system?

**Comment by ac3nl** [comment]
you can do that yourself
"clone the wonderswan" emu map, and adjust the directorties where they should point to
make an new map in de roms dir, SAME NAME as the CLONED wonderswan map

**Reply by Shadow322** [reply]
Guide on how?

**Comment by ac3nl** [comment]
[onion.zip](https://github.com/OnionUI/Onion/files/10208406/onion.zip)

here is my WonderSwan Color setup
copy all the Wonderswan Color Roms to Roms/WSC map, leave the others is the Roms/WS map

**Reply by Shadow322** [reply]
thanks what about one for separating NGP and NGPC?

**Comment by ac3nl** [comment]
[onion.zip](https://github.com/OnionUI/Onion/files/10208781/onion.zip)



Here are all my custom dirs, so you don't have to ask them seperatly

**Reply by ac3nl** [reply]
You’re welcome….
